### PR TITLE
python3Packages.iterm2: init at 1.14

### PR DESCRIPTION
**Pull request by jeremyschlatter**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3514,6 +3514,12 @@
     githubId = 1608697;
     name = "Jens Binkert";
   };
+  jeremyschlatter = {
+    email = "github@jeremyschlatter.com";
+    github = "jeremyschlatter";
+    githubId = 5741620;
+    name = "Jeremy Schlatter";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:3522]: Please make this into a separate commit:

````
maintainers: add jeremyschlatter
```

File: pkgs/development/python-modules/iterm2/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "089pln3c41n6dyh91hw9gy6mpm9s663lpmdc4gamig3g6pfmbsk4";
+  };
+
+  propagatedBuildInputs = [ protobuf websockets ];
+
+  # The tests require pyobjc. We can't use pyobjc because at
+  # time of writing the pyobjc derivation is disabled on python 3.
+  # iterm2 won't build on python 2 because it depends on websockets
+  # which is disabled below python 3.3.
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/iterm2/default.nix:21]: If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jeremyschlatter) [reply maintainers/maintainer-list.nix:3522]: Done.

File: pkgs/development/python-modules/iterm2/default.nix (hunk 2 above)
Comment (jeremyschlatter) [reply pkgs/development/python-modules/iterm2/default.nix:21]: Done.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 90199 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary><br>- python37Packages.iterm2<br>- python38Packages.iterm2</details>

**Comment by jonringer** [comment]
@jeremyschlatter congrats on your first nixpkgs PR :)

**Comment by jeremyschlatter** [comment]
@jonringer thanks! :)
